superset/jinja_context.py

```diff
@@ -104,13 +104,11 @@ class ExtraCache:
     # Regular expression for detecting the presence of templated methods which could
     # be added to the cache key.
     regex = re.compile(
-        r"\{\{.*("
-        r"current_user_id\(.*\)|"
+        r"(current_user_id\(.*\)|"
         r"current_username\(.*\)|"
         r"current_user_email\(.*\)|"
         r"cache_key_wrapper\(.*\)|"
-        r"url_param\(.*\)"
-        r").*\}\}"
+        r"url_param\(.*\))"
```

Comment:
nit: we could be more strict and make sure we have:
{{ ... current_username() }} or {% current_username() %}

Reply:
I agree with Daniel. We should check for both valid cases `{{` and `{%` to avoid false positives (although I admit they're going to be rare).